extensions/fairifier-rdf-extension/dtls/fairifier/ListRdfSkeletonsCommand.java

```diff
@@ -12,47 +13,49 @@
 
 import com.google.refine.commands.Command;
 
-public class ListRdfSkeletonsCommand extends Command{
+public class ListRdfSkeletonsCommand extends Command {
+
     private static final RdfSkeletonService rdfSkeletonService = new FileSystemRdfSkeletonImpl();
-    
+
     @Override
-    public void doPost(HttpServletRequest req, HttpServletResponse res) throws ServletException, IOException {
+    public void doPost(HttpServletRequest req, HttpServletResponse res)
+            throws ServletException, IOException {
         res.setCharacterEncoding("UTF-8");
         res.setHeader("Content-Type", "application/json");
         JSONWriter writer = new JSONWriter(res.getWriter());
```

Comment:
Can we get around this and the `writer.key()` and `writer.value()` verbosity by using jackson and writing the metadata to the response directly?

Reply:
i could but it would be a quite elaborate sollution, org.json and jackson are different libraries, i could map the Model POJO onto a JSONObject provided by org.json, but by default you can not use this writer with foreign objects.

Reply:
OK, let's leave it for now.